### DISCUSSION TITLE
Ragna 0.2.0 release tracker

### DISCUSSION THREAD

**Discussion by pmeier** [Quansight/ragna#306] [Announcements]
_This post will be updated regularly until `0.2.0` is released_

---

Feb 2, 2024

We are closing in on the `0.2.0` release. The main two features for this release is the removal of the task queue and as a direct result the ability to stream answers from the assistants. If you want to test these features, we already published some alpha versions to PyPI. Install them with

```bash
$ pip install --upgrade --pre 'ragna[all]'
```

Apart from these larger features, the release will also bring a lot of smaller patches. A lot of it is merged already, but there are still a [few open items](https://github.com/Quansight/ragna/milestone/4). Four of them are good starting issues for new contributors.

---

March 5, 2024

After some delay, we finally published the first release candidate `0.2.0rc0` today. Install it with `pip install --pre 'ragna[all]'`. You can find the preliminary release notes [here](https://ragna--278.org.readthedocs.build/en/278/references/release-notes/#version-020).

From here on I expect mostly bug fixes and documentation updates until the final release.

**Comment by NetDevAutomate** [comment]
This release seems to work really well, especially with ragna-aws! Fantastic!!

**Comment by pmeier** [comment]
0.2.0 is out: https://github.com/Quansight/ragna/releases/tag/v0.2.0
